io.github.rahatzamancse.ProtonGameSaves.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.github.rahatzamancse.ProtonGameSaves",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "proton-game-saves",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--own-name=io.github.rahatzamancse.ProtonGameSaves",
+        "--filesystem=~/.local/share/Steam:rw",
+        "--filesystem=~/.steam:rw"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
+        ],
```

Comment:
not allowed, how do you miss the giant text in the requirements page...

Reply:
Sorry, this is my first submission. Removing it.